driver/csiplugin/gpfs.go

```diff
@@ -42,6 +42,16 @@ const (
 	VOLCOPY_JOB_RUNNING     = 5
 	VOLCOPY_JOB_COMPLETED   = 6
 	VOLCOPY_JOB_NOT_STARTED = 7
+
+	STORAGECLASS_CLASSIC  = "0"
+	STORAGECLASS_ADVANCED = "1"
```

Comment:
should we renamed STORAGECLASS_ADVANCED to something lik STORAGECLASS_CONSISTENTGROUP because this is more specific to consistentcy group while other feature should be enabled for basic type of storageclass also?

Reply:
One reason for name is since we take input as "scType = "advanced". The name was decided based on agreement.

Reply:
@deeghuge any thoughts here ?

Reply:
Changed the SC parameter to `type`